exporter/opentelemetry-exporter-zipkin/tests/test_zipkin_exporter.py

```diff
@@ -440,8 +457,66 @@ def test_export_json_max_tag_length(self):
         _, kwargs = mock_post.call_args  # pylint: disable=E0633
 
         tags = json.loads(kwargs["data"])[0]["tags"]
-        self.assertEqual(len(tags["k1"]), 128)
-        self.assertEqual(len(tags["k2"]), 50)
+
```

Comment:
To avoid these long testing sequences, may I suggest:

``` python
from functools import partial
from json import dumps
...
partial_dumps = partial(dumps, separators=(",", ":"))
```